.pylintrc

```diff
@@ -95,144 +120,229 @@ reports=yes
-argument-rgx=[a-z_][a-z0-9_]{0,30}$
+# Regular expression matching correct class names
+class-rgx=[A-Z_][a-zA-Z0-9]+$
 
-# Regular expression which should only match correct variable names
-variable-rgx=[a-z_][a-z0-9_]{1,30}$
+# Naming hint for class names
+class-name-hint=[A-Z_][a-zA-Z0-9]+$
 
-# Regular expression which should only match correct attribute names in class
-# bodies
-class-attribute-rgx=([A-Za-z_][A-Za-z0-9_]{2,30}|(__.*__))$
+# Regular expression matching correct module names
+module-rgx=(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$
 
-# Regular expression which should only match correct list comprehension /
-# generator expression variable names
-inlinevar-rgx=[A-Za-z_][A-Za-z0-9_]*$
+# Naming hint for module names
+module-name-hint=(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$
 
-# Good variable names which should always be accepted, separated by a comma
-good-names=i,j,k,ex,Run,_,f
+# Regular expression matching correct method names
+method-rgx=(test_[a-z0-9_]+|[a-z_][a-z0-9_]{2,40})$
 
-# Bad variable names which should always be refused, separated by a comma
-bad-names=foo,bar,baz,toto,tutu,tata,tmp,tmp2
+# Naming hint for method names
+method-name-hint=([xyz]|test_[a-z_][a-z0-9_]*|[a-z_][a-z0-9_]{2,40})$
 
 # Regular expression which should only match function or class names that do
 # not require a docstring.
-no-docstring-rgx=__.*__
+no-docstring-rgx=^_
 
 # Minimum line length for functions/classes that require docstrings, shorter
 # ones are exempt.
 docstring-min-length=-1
 
 
-[MISCELLANEOUS]
+[ELIF]
 
-# List of note tags to take in consideration, separated by a comma.
-notes=FIXME,XXX,TODO
+# Maximum number of nested blocks for function / method body
+max-nested-blocks=5
+
+
+[SIMILARITIES]
+
+# Minimum lines number of a similarity.
+min-similarity-lines=4
+
+# Ignore comments when computing similarities.
+ignore-comments=yes
+
+# Ignore docstrings when computing similarities.
+ignore-docstrings=yes
+
+# Ignore imports when computing similarities.
+ignore-imports=no
+
+
+[LOGGING]
+
+# Logging modules to check that the string format arguments are in logging
+# function parameter format
+logging-modules=logging
+
+
+[VARIABLES]
+
+# Tells whether we should check for unused import in __init__ files.
+init-import=no
+
+# A regular expression matching the name of dummy variables (i.e. expectedly
+# not used).
+dummy-variables-rgx=(_+[a-zA-Z0-9]*?$)|dummy
+
+# List of additional names supposed to be defined in builtins. Remember that
+# you should avoid to define new builtins when possible.
+additional-builtins=
+
+# List of strings which can identify a callback function by name. A callback
+# name must start or end with one of those strings.
+callbacks=cb_,_cb
+
+# List of qualified module names which can have objects that can redefine
+# builtins.
+redefining-builtins-modules=six.moves,future.builtins
 
 
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=120
+max-line-length=100
```

Comment:
Isn't 100 a little bit arbitrary? (AFAIK 80 is the standard, 120 is the company-standard, but I don't know where 100 comes from.)

Reply:
This was an accident when upgrading .pylintrc, should be 120, the same as pycodestyle has.